_layouts/quality_estimation.html

```diff
@@ -0,0 +1,124 @@
+---
+layout: default
+parent: Quality estimation
+---
+
+<h1>
+  {{ page.title }}
+</h1>
+<h2>
+  {{ page.description }}
+</h2>
+
+{{ content }}
+
+<hr/>
+
+<p>
+  {% if page.supported_languages.size > 0 %}
+   <strong>{{ page.title }}</strong> supports <a href="#language-support">{{ page.supported_languages | size }} languages</a>.
+  {% else %}
+    The <strong>{{ page.title }}</strong> supports an unknown number of languages.
+  {% endif %}
+</p>
+
+<p>
+  {% if page.customisation %}
+  <a href="/customisation">Customisation</a> is supported.
+  {% endif %}
+</p>
+
+
+<blockquote>
+
+  <dl>
+
+    <dt>Language support</dt>
+  {% if page.supported_languages.size == 0 %}
+    <dd><a href="#language-support">Unknown number of languages</a></dd>
+  {% else %}
+    <dd><a href="#language-support">{{ page.supported_languages | size }} languages</a></dd>
+  {% endif %}
+
+  
+  <dt>Integrations</dt>
+  {% if page.only_compatible_tms %}
+  <dd><a href="/{{ page.only_compatible_tms }}">{{ page.only_compatible_tms|capitalize }}</a></dd>
+  {% else %}
+  <dd>Any translation management system (API)</dd>
+  {% endif %}
+  
+  <dt>APIs</dt>
+  {% if page.only_compatible_mt_api %}
```

Comment:
Is there a way that we can have the TMS or API ´name´here, instead of the ´id´?

![Screenshot 2023-09-17 at 15 15 25](https://github.com/machinetranslate/machinetranslate.org/assets/33723604/7bbf9bce-09ce-4a3b-84ac-fca4839ac781)